### PR TITLE
8328730: Convert java/awt/print/bug8023392/bug8023392.html applet test to main

### PR DESCRIPTION
**Pull request by DamonGuy**
Convert java/awt/print/bug8023392/bug8023392.html applet test to main using PassFailJFrame

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8328730](https://bugs.openjdk.org/browse/JDK-8328730): Convert java/awt/print/bug8023392/bug8023392.html applet test to main (**Bug** - P4)

### Reviewers
 * [Phil Race](https://openjdk.org/census#prr) (@prrace - **Reviewer**) ⚠️ Review applies to [72fe9b13](https://git.openjdk.org/jdk/pull/18458/files/72fe9b131da519de1c44417fd470e529e2a39fcc)
 * [Harshitha Onkar](https://openjdk.org/census#honkar) (@honkar-jdk - **Reviewer**) ⚠️ Review applies to [72fe9b13](https://git.openjdk.org/jdk/pull/18458/files/72fe9b131da519de1c44417fd470e529e2a39fcc)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/18458/head:pull/18458` \
`$ git checkout pull/18458`

Update a local copy of the PR: \
`$ git checkout pull/18458` \
`$ git pull https://git.openjdk.org/jdk.git pull/18458/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 18458`

View PR using the GUI difftool: \
`$ git pr show -t 18458`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/18458.diff">https://git.openjdk.org/jdk/pull/18458.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/18458#issuecomment-2015559941)

### CODE REVIEW

File: test/jdk/java/awt/print/bug8023392.java [hunk 1]
```diff
@@ -84,15 +113,15 @@ public SimplePrint2() {
                 String s = "3a) a b c d e                                     ";
                 @Override
                 protected void paintComponent(Graphics g) {
-                    sun.swing.SwingUtilities2.drawChars(this, g, s.toCharArray(),
+                    SwingUtilities2.drawChars(this, g, s.toCharArray(),
```
Comment (prsadhuk) [inline test/jdk/java/awt/print/bug8023392.java:116]: cant we use `Graphics.drawChars` instead in all places `SwingUtilities2.drawChars` is used?

Comment (DamonGuy) [reply test/jdk/java/awt/print/bug8023392.java:116]: `Graphics.drawChars` isn't a static method, which will require additional changes to this test. Is it preferred to use `Graphics` over the `SwingUtilities2` version?

Comment (prrace) [reply test/jdk/java/awt/print/bug8023392.java:116]: This test is specifically about testing the spacing of text in Swing printing.
I think this SwingUtilities call isn't just "the way he happens to be printing text",
This call is the very thing under test ! And note that it takes a Component parameter
which affects how that call renders the text. It is used to figure out the FRC to use.
So changing this to use Graphics would be a problem.

**Review by prrace — APPROVED** [review]

File: test/jdk/java/awt/print/bug8023392.java [hunk 2]
```diff
@@ -177,111 +208,4 @@ public int print(Graphics graphics,
...
-    }//printInstructions()
-
-    //DO NOT call this directly, go through Sysout
-    public void displayMessage(String messageIn) {
-        messageText.append(messageIn + "\n");
-    }
-
-}// TestDialog  class
-
+}
```
Comment (honkar-jdk) [inline test/jdk/java/awt/print/bug8023392.java:211]: EOF newline missing.

**Review by honkar-jdk — APPROVED** [review]
LGTM

**Pull request closed without merging**